bigtable/google/cloud/bigtable/row.py

```diff
@@ -17,7 +17,9 @@
 
 import struct
 
+import functools
 import six
+import grpc
```

Comment:
grpc goes before six

imports are in three groups: standard library, third-party, and package-local. Each group should be in alphabetical order.